docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -152,16 +162,11 @@ If you wish to use output from an `OpenScPCA-nf` analysis module in your module,
 We have provided a [`download-results.py` script](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/download-results.py) that you can use to download results from a module that has been run in `OpenScPCA-nf`.
 
 We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
-
-You can list all the options for `download-results.py` by running the following from the root directory of the repository:
-
-```sh
-./download-results.py --help
-```
+All examples below assume you are running the script from the root of the repository.
 
 !!! tip "Log into AWS CLI before running the script"
-    Before running this script locally, you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
-    To do this, run the following commands in terminal before running the `results-download.py` script, and follow instructions to log in.
+    Before running this script locally (i.e., not from an [ALSF-provided virtual computer](../../aws/index.md#lightsail-for-research-virtual-computing-with-aws)), you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
```

Comment:
```suggestion
    Before running this script locally (i.e., not using [Lightsail for Research](../../aws/index.md#lightsail-for-research-virtual-computing-with-aws)), you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
```

Reply:
Huh, I thought I got this... Oh well, thanks!